README.md

```diff
@@ -690,6 +691,40 @@ Here is a sample app.config file that uses [log4net](http://logging.apache.org/l
   </log4net>
 ```
 
+Easy logging
+------------
+
+The Easy logging feature allows you to change log level of all driver's classes and add extra file appender for logs from driver's classes in runtime.
+This feature was introduced to make tracing of driver's logs easier.
+The feature is activated by a config file which can be:
+- provided in connection string as `CLIENT_CONFIG_FILE` parameter (eg. `ACCOUNT=test;USER=test;PASSWORD=test;CLIENT_CONFIG_FILE=/some-path/client_config.json;`)
+- provided as environmental variable called `SF_CLIENT_CONFIG_FILE` (eg. `export SF_CLIENT_CONFIG_FILE=/some-path/client_config.json`)
+- found in the driver location by searching for `sf_client_config.json` file
+- found in the home location by searching for `sf_client_config.json` file
+- found in temp directory location by searching for `sf_client_config.json` file
+
+The search for a config file is executed in the order listed above.
+
+To minimize the number of searches for a configuration file it is executed only:
+- for the first connection
+- for the first connection with `CLIENT_CONFIG_FILE` parameter.
```

Comment:
Currently it is really for the first. Do you think it's worth to change the logic to execute it for every unique parameter?

Reply:
At least if they resolve to the same absolute path maybe. Otherwise if we have more than one pool, the first one wins. It can be even worse - it can be non deterministic. What do you think?

Reply:
Making it configure for each unique parameters makes it even more non deterministic.
I like my solution more because it addresses such use cases:
1. the client places configuration in one of predefined folders, runs his application, disk search is done only once, logging is working as configured in the config file
2. the client has problem only with one operation in his application, so he adds CLIENT_CONFIG_FILE param to connection string of one of the operations. The disk search would be done only once. The configured logs will start from the connection the client wants to look at.

What use case covers configuring it for every unique parameter?

Reply:
By `parameter` I meant `CLIENT_CONFIG_FILE`, not any parameter. Does it make more sense now?

Reply:
we discussed that we will not change the behaviour